index.html

```diff
@@ -0,0 +1,273 @@
+    <header>
+        <div class="logo-container">
+            <a href="#">Welcome</a>
+        </div>
+        <nav>
+            <ul>
+                <li><a class="nav-menu" href="#works-section">Portfolio</a></li>
+                <li><a class="nav-menu" href="#about-section">About</a></li>
+                <li><a class="nav-menu" href="#contact-section">Contact</a></li>
+            </ul>
+            <button type="button" id="hamburger"><img src="icon-img/hamburger.svg" alt=""></button>
+        </nav>
+    </header>
+
+    <!-- Main content starts here -->
+    <section class="headline">
+            <div class="heading">
+                <h1>Hey there. I'm Abdallah</h1>
+                <h2>I'm a software developer</h2>
+                <p> I can help you build a product, feature or website. Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don't hesitate to contact me.</p>
+            </div>
+
+            <ul>
+                <li><a class="media-link" href="https://twitter.com/abdallah_cu"><img src="icon-img/twitter.svg"
+                            alt="twitter logo"></a></li>
+                <li><a class="media-link"
+                        href="https://www.linkedin.com/in/abdallah-cuma-5a7617274/"><img
+                            src="icon-img/linkedin-icon.svg" alt="linked-in logo"></a></li>
+                <li><a class="media-link" href="https://medium.com"><img src="icon-img/medium.svg"
+                            alt="medium logo"></a></li>
+                <li><a class="media-link" href="https://github.com/AbdallahJumah95?tab=repositories"><img src="icon-img/github.svg"
+                            alt="github logo"></a></li>
+                <li><a class="media-link" href="https://angel.co/"><img src="icon-img/angellist.svg"
+                            alt="angellist logo"></a>
+                </li>
+            </ul>
+            <a href="#works-section" class="action-btn-sec"><img src="icon-img/chevron-down.svg" alt="chevron down action button"></a>
+        </section>
+    
+        <!-- Works Section  -->
+        <section id="works-section">
+            <div class="container">
+                <h2>My Recent Works</h2>
+                <img src="icon-img/indicator.svg" alt="horizontal line indicator">
+                
+                <div class="works-grid-container">
+                    <div class="works-grid-card">
+                        <div class="works-card-meta">
+                            <h3>Multi-Post Stories Gain + Glory</h3>
+                            <ul class="work-categories">
+                                <li>Ruby on rails</li>
+                                <li>css</li>
+                                <li>JavaScript</li>
+                                <li>html</li>
+                            </ul>
+        
+                            <a href="#" class="action-btn">See Project</a>
```

Comment:
- [ ] Great job on your portfolio so far! To make it even better, I suggest adding `hover, active, and disabled` effects to your buttons. **This will give your portfolio a more professional and polished look**, and help bring it closer to the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&t=TQVVV1GU9LawHn3z-0). Please consider adding these effects, and let me know if you have any questions. 🙌

  ![image](https://user-images.githubusercontent.com/107952724/236656335-77eb8332-285a-438d-9101-588a03ca7f40.png)